File: app/src/main/java/com/constraint/vagabond/data/local/RecAreaLocalDataStore.java
package com.constraint.vagabond.data.local;

import com.constraint.vagabond.data.RecAreaDataSource;
import com.constraint.vagabond.data.entities.RecreationalArea;

import java.util.List;

public class RecAreaLocalDataStore implements RecAreaDataSource {

  @Override
  public void save(List<RecreationalArea> recreationalAreaList) {}

  @Override
  public List<RecreationalArea> getAreas() {
    return null;
  }

  @Override
  public RecreationalArea getArea(String rec_area_id) {
    return null;
  }
}

File: app/src/main/java/com/constraint/vagabond/data/local/RecAreaLocalDataStore.kt
package com.constraint.vagabond.data.local

import com.constraint.vagabond.data.RecAreaDataSource
import com.constraint.vagabond.data.entities.RecreationalArea

class RecAreaLocalDataStore : RecAreaDataSource {

    override val areas: List<RecreationalArea>
        get() = TODO("not implemented") //To change initializer of created properties use File | Settings | File Templates.

    override fun save(recreationalAreaList: List<RecreationalArea>) {}

    override fun getArea(rec_area_id: String): RecreationalArea? {
        return null
    }
}
